src/styles/sidebar/components/FilterSelect.scss

```diff
@@ -2,6 +2,12 @@
 @use "../../variables" as var;
 
 .FilterSelect {
+  &__menu {
```

Comment:
Something to note for later is that this is going to assert this max-height on all subsequent filters we add—that is, we're likely going to add a "document" filter in short order. That might be just fine, but we may need to differentiate for the user-filter specifically at some point. Nothing to do now!

Reply:
Yes, we might need to be more specific here. Lets address that when we make the doc filter.